src/emcpy/stats.py

```diff
@@ -193,3 +195,33 @@ def get_weighted_mean(data, weights, axis=None):
         'data and weights mis-match array size')
 
     return _np.average(data, weights=weights, axis=axis)
+
+
+def get_linear_regression(data1, data2):
+    """
+    Calculate linear regression between two sets of data.
+    Fits a linear model with coefficients to minumize the
+    residual sum of squares between the observed targets
+    in the dataset, and the targets predicted by the linear
+    approximation.
+    INPUT
+        data1, data2 - data to calculate linear
+                       regression (array like)
+    OUTPUT
+        y_pred - predicted y values of from calculation (float)
+        r_sq - r squared value (float)
+        intercept - intercept from slope intercept equation for
+                    y_pred (float)
+        slope - slope from slope intercept equation for y_pred
+                (float)
+    """
+    x = np.array(data1).reshape((-1, 1))
+    y = np.array(data2)
```

Comment:
what is the purpose of this here?
Should'nt the inputs to this method already take care of this?

Reply:
Good point. No need for the np.array(). However, to use the linear regression method, x does need to be reshaped appropriately so that should stay.